README.md

```
@@ -1,146 +1,119 @@
-I am using [`dotdrop`](https://github.com/deadc0de6/dotdrop) to manage dotfiles and [`ansible`](https://github.com/ansible/ansible) to set things up. Steps:
+1. Clone the repository:
 
-1. Clone this repo with: `git clone https://github.com/wilfriedago/dotfiles ~/.dotfiles`
-2. `cd ~/.dotfiles/`
-3. Run the following commands to install the necessary tooling:
+```zsh
+git clone https://github.com/wilfriedago/dotfiles.git ~/.dotfiles
+cd ~/.dotfiles
+```
+
+2. Install dependencies and set up environment:
```

markdownlint-cli2 (0.17.2)

Ordered list item prefix
Expected: 1; Actual: 2; Style: 1/1/1

(MD029, ol-prefix)